packages/terra-date-picker/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
+* Lock in version of popper to prevent date picker from pointing to an incorrect position when invoked. https://github.com/cerner/terra-core/issues/1402
```

Comment:
Leave the link out of the changelog. I don't think it's necessary, and we can easily trace back to this issue through git history.

Reply:
I'd prefer to keep the link, I think it makes it easier for consumers to find out about the change